ansible/roles/stack-sunbird/templates/sunbird_player.env

```diff
@@ -72,3 +72,6 @@ sunbird_trampoline_android_keycloak_client_id={{sunbird_trampoline_android_keycl
 sunbird_trampoline_android_keycloak_secret={{sunbird_trampoline_android_keycloak_secret}}
 sunbird_android_keycloak_client_id={{sunbird_android_keycloak_client_id}}
 
+#Release-2.3.5
+
+ sunbird_portal_merge_auth_server_url={{sunbird_portal_merge_auth_server_url | d('')}}
```

Comment:
Please change the second part to {{sunbird_subdomain_keycloak_base_url | d('')}}

Reply:
Have Changed